### ISSUE TITLE
Angry Birds: Trilogy – Random pigs/birds' eyes flicker or blink too fast

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2144]
I think the new CPU JIT introduced this bug as when I switch back to interpreter this issue is not noticeable. (Or could be just too slow to be noticable) Since I did the counting with the low FPS taking into account, I can most likely confirm it does not happen in interpreter mode.
It could be due to some timing issues in _Dynarmic_

Most of the time the pigs/birds eyes flickers happen randomly between all numbers of birds/pigs on-screen at the top of the 3DS screen. Sometimes it is just one or two or even three happen simultaneously (less likely). Some times it does not flicker but instead blink too fast like blinking every second which is not the real behavior shown on the actual console itself.

During this scene the emulator's FPS is shown at around 75 (using Dxtory).
![Flickering Issue](https://cloud.githubusercontent.com/assets/18496868/19774485/ce6940fe-9c9f-11e6-9813-e361e493e77f.png)

**Steps to reproduce this issue:**
1. Just open the game and pull the slingshot once until you reach the scene as shown above.
2. If you don't see the flickering just reset the game will most likely do the trick.

**_Note:_** _Do not pull the slingshot once you reach the scene above as doing so will remove the pig from showing up next time (To revert this state, just remove the save file.)_

**Comment by Dragios** [comment]
Might be related to #2130

**Comment by Dragios** [comment]
Fixed by #2234

**Comment by Subv** [comment]
Probably 
````
14eb70d VFP: Fixed the VCVT behavior when converting from unsigned 32-bit values. (#51)
````
It's the only VFP-related change in there.
